corehq/messaging/scheduling/tests/test_content.py

```diff
@@ -50,11 +50,12 @@ def setUpClass(cls):
         cls.sms_translations = get_or_create_sms_translations(cls.domain)
         cls.sms_translations.set_translations('es', {})
         cls.sms_translations.save()
-        cls.mobile_user = CommCareUser.create(cls.domain, 'mobile', 'abc', None, None, device_id='test_dev')
+        cls.mobile_user = CommCareUser.create(cls.domain, 'test-mobile', 'abc', None, None, device_id='test_dev')
```

Comment:
strange, why was this needed?

Reply:
I added this as an additional safety however TBH this is not needed. Resolved in 9fcd1886bfd67c312fac1168a1166406ac295612